Content.Server/Zombies/PendingZombieComponent.cs

```diff
@@ -26,4 +25,17 @@ public sealed class PendingZombieComponent : Component
     // Scales damage over time.
     [DataField("infectedSecs")]
     public int InfectedSecs;
+
+    // Infection warnings are shown as popups, times are in seconds.
+    //   -ve times shown to initial zombies (once timer counts from -ve to 0 the infection starts)
+    //   +ve warnings are in seconds after being bitten
+    [DataField("infectionWarnings")]
+    public Dictionary<int, string> InfectionWarnings = new()
+    {
+        {-45, "zombie-infection-warning"},
+        {-30, "zombie-infection-warning"},
```

Comment:
does infectedSecs ever get below 0? i dont see anything substracting from it ever

Reply:
It initialises below zero to say -1200 (infection forced in 20 minutes) by the `ZombieGameRule`

Reply:
oh, forgot to check the prototype, disregard then